main.cpp

```diff
@@ -384,96 +314,311 @@ each property for UDTs 5 - 9.
 Step 3 is written below UDTs 5 - 9
 */
 
+/*
+===================
+Part 1d assignment: Step 3
+===================
+You've just defined 10 UDTs!
+4 of them are unrelated (UDTs 1-4).
+5 of them form the sub-objects that make up the 10th UDT. 
+
+MOVE THEM BELOW this block comment and put them in numerical order 
+(1 - 10) starting on line 473
+    DO NOT COPY. 
+    I do not want to see 2 copies of your Plain-English UDTs.  
+    I only want to see the 10 UDTs written BELOW this block comment, in numerical order (1 - 10).
+
+After you have MOVED your 10 UDTs, send me a DM with your pull request link.
+I will review the pseudo-code that you have written.
+*/
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+/*
+Thing 1)
+Kitchen
+5 properties:
+    1) number of pans (int)
+    2) number of pots (int)
+    3) size in square meters (float)
+    4) wall color (std::string)
+    5) number of plates (int)
+3 things it can do:
+    1) prepare food
+    2) wash dishes
+    3) store food
+*/
+
+/*
+Thing 2)
+Computer
+5 properties:
+    1) is a laptop (bool)
+    2) ssd storage in gigabytes (int)
+    3) ram in gigabytes (int)
+    4) screen size in inches (float)
+    5) operating system (std::string)
+3 things it can do:
+    1) browse the internet
+    2) play video game
+    3) write text document
+ */
+
+/*
+Thing 3)
+Bicycle
+5 properties:
+    1) amount of gears (int)
+    2) type of valve on tire (std::string)
+    3) has back brake (bool)
+    4) frame color (std::string)
+    5) type of saddle (std::string)
+3 things it can do:
+    1) shift gears
+    2) turn
+    3) accelerate
+ */
+
+/*
+Thing 4)
+Digital Mixing Desk
+5 properties:
+    1) number of channels (int)
+    2) audio network protocol (std::string)
+    3) number of faders (int)
+    4) has touch screen (bool)
+    5) weight in kilograms (float)
+3 things it can do:
+    1) mix audio
+    2) record audio
+    3) save current scene
+ */
+
 /*
 Thing 5)
+Cockpit
 5 properties:
-    1)
-    2)
-    3)
-    4)
-    5)
+    1) number of seats (int)
+    2) number of windows (int)
+    3) number of displays (int)
+    4) size in square meters (float)
+    5) number of control wheels (int)
 3 things it can do:
-    1)
-    2)
-    3)
+    1) change aircraft speed
+    2) display current altitude
+    3) use autopilot
  */
 
 /*
 Thing 6)
+Passenger Area
 5 properties:
-    1)
-    2)
-    3)
-    4)
-    5)
+    1) number of economy class seats (int)
+    2) number of business class seats (int)
+    3) location of life vests (std::string)
+    4) engine noise level in db (float)
+    5) has adjustable seats (bool)
```

Comment:
```
    Notice that I did not use "has a vacuum cleaner" or "Has 
    eco-friendly cleaning supplies" as one of the properties.
    
    Writing 'has a ___" checks whether or not your object **has the 
    ability to do something**.
    Instead, I wrote "number of vacuum cleaners" and "number of 
    eco-friendly cleaning supplies".  
    
    These are specific objects or amounts. 
    
    In C++ terms, this means to I want you to avoid using 'bool' 
    (has a) as a member variable type.
    Instead, prefer the other primitive types.
```
- [ ] revise